components/slider/index.css

```diff
@@ -640,97 +641,72 @@ governing permissions and limitations under the License.
-      .spectrum-Slider-track:first-child {
-        &::before {
-          background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
-        }
-      }
+  &.spectrum-Slider--filled {
+    .spectrum-Slider-track:first-child::before {
+      background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
     }
+  }
 
-    .spectrum-Slider-fill {
-      &::before {
-        background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
-      }
+  .spectrum-Slider-fill {
+    &::before {
+      background: var(--highcontrast-slider-track-fill-color-disabled, var(--mod-slider-track-fill-color-disabled, var(--spectrum-slider-track-fill-color-disabled)));
     }
+  }
 
-    .spectrum-Slider-ramp {
-      /* Slider handle turns transparent in high contrast mode for ramp */
-      @media (forced-colors: active) {
-        & + .spectrum-Slider-handle {
-            fill: ButtonFace;
-            background-color: ButtonFace;
-          }
-      }
-
-      path {
-        fill: var(--highcontrast-slider-ramp-track-color-disabled, var(--mod-slider-ramp-track-color-disabled, var(--spectrum-slider-ramp-track-color-disabled)));
-      }
+  .spectrum-Slider-ramp {
+    path {
+      fill: var(--highcontrast-slider-ramp-track-color-disabled, var(--mod-slider-ramp-track-color-disabled, var(--spectrum-slider-ramp-track-color-disabled)));
     }
+  }
 
-    .spectrum-Slider-tick {
-      &:after {
-        background-color: var(--highcontrast-slider-tick-mark-color-disabled, var(--mod-slider-tick-mark-color-disabled, var(--spectrum-slider-tick-mark-color-disabled)));
-      }
+  .spectrum-Slider-tick {
+    &::after {
+      background-color: var(--highcontrast-slider-tick-mark-color-disabled, var(--mod-slider-tick-mark-color-disabled, var(--spectrum-slider-tick-mark-color-disabled)));
     }
+  }
 
-    &.spectrum-Slider--range {
-      .spectrum-Slider-track {
-        &:not(:first-of-type):not(:last-of-type) {
-          &:before {
-            background: var(--highcontrast-slider-track-color-disabled, var(--mod-slider-track-color-disabled, var(--spectrum-slider-track-color-disabled)));
-          }
-        }
-      }
+  &.spectrum-Slider--range {
+    .spectrum-Slider-track:not(:first-of-type):not(:last-of-type)::before {
+      background: var(--highcontrast-slider-track-color-disabled, var(--mod-slider-track-color-disabled, var(--spectrum-slider-track-color-disabled)));
     }
   }
 }
 
-
 @media (forced-colors: active) {
 
   .spectrum-Slider {
-    --spectrum-slider-track-color: GrayText;
-    --spectrum-slider-track-fill-color: GrayText;
-
-    --spectrum-slider-track-color-disabled: GrayText;
-    --spectrum-slider-track-fill-color-disabled: GrayText;
```

Comment:
Just calling these out to be sure they were intentionally removed.

Reply:
They were intentionally removed because they were being overwritten anyway by the `highcontrast` variables below this